client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -208,7 +208,7 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 		const { getForwardUrl, hasBusinessAddress, siteId, translate } = this.props;
 
 		return (
-			<div className="steps__main">
+			<div className="steps__main" data-e2e-type="business-address">
```

Comment:
cc @Automattic/flowpatrol - is this a common pattern, or would it be easier if we introduce a classname like `steps__business-address` to each step instead?